### ISSUE TITLE
How to use other datasets to training?

### ISSUE THREAD

**Issue by shihe407** [ArjunNarayanan/LinFlo-Net#8]
Hi, it's me again.
If I want to use other datasets (e.g. ACDC or some private dataset) for training and testing, does the program have any other requirements for these? Are there changes need to be made to the preprocessing steps?
I tried the ACDC dataset but ACDC is 4D data, I extracted arbitrary frames but found it can't start training.

**Comment by ArjunNarayanan** [comment]
Yes, you’ll need to extract frames from 4D data. Were you able to run data
augmentation on the data?

Arjun Narayanan

**Comment by shihe407** [comment]
Thanks for your quick reply.
I couldn‘t to use ACDC to run data augmentation.
Do I need to use the project's preprocessing code to preprocess the data before augmentation?

**Comment by ArjunNarayanan** [comment]
There’s a section in the Readme for data augmentation, did you try that?

You don’t need to do data aug to train the model, but it usually produces
better results.

Arjun Narayanan

**Comment by shihe407** [comment]
I have tried the MMWHS dataset to data augmentation and I have trained the model with MMWHS.
Now I want to use other dataset but I meet some problems.
One more thing, I find the ACDC images' view is a little different with MMWHS's, will this work rightly?

**Comment by ArjunNarayanan** [comment]
Yes the view matters so I would try to get the ACDC images consistent with
what you see in MMWHS.

Arjun Narayanan

**Comment by shihe407** [comment]
Thank you, I also tried but found it difficult to align the MMWHS perspective with the ACDC perspective.
Thanks for your trying.
